imageproxy.go

```diff
@@ -324,6 +325,16 @@ func hostMatches(hosts []string, u *url.URL) bool {
 		if strings.HasPrefix(host, "*.") && strings.HasSuffix(u.Host, host[2:]) {
 			return true
 		}
+		// Checks whether the host in u is an IP
+		if ip := net.ParseIP(u.Host); ip != nil {
+			// Checks whether our current host is a CIDR
+			if _, ipnet, err := net.ParseCIDR(host); err == nil {
+				// Checks if our host contains the IP in u
+				if ok := ipnet.Contains(ip) == true; ok {
```

Comment:
any reason you did this rather than simply:

```go
if ipnet.Contains(ip) {
  return true
}
```

Reply:
Nope, just overlooked it -- sorry. Fixed in f646549b81a69f8d783fdd5ac6a4ee76dade0ea4.